docs/queries.md

```diff
@@ -241,6 +241,91 @@ const queryRes = await Person.findOne({ _id: idString });
 const aggRes = await Person.aggregate([{ $match: { _id: idString } }])
 ```
 
+<h3 id="sorting"><a href="#sorting">Sorting</a></h3>
+
+[Sorting](/docs/api.html#query_Query-sort) is how you can ensure you db requests come back in the desired order.
+
+```javascript
+const personSchema = new mongoose.Schema({
+  age: Number
+});
+
+const Person = mongoose.model('Person', personSchema);
+for (let i = 0; i < 10; i++) {
+  await Person.create({ age: i });
+}
+
+await Person.find().sort({ age: -1 }); // returns age starting from 10 as the first entry
+await Person.find().sort({ age: 1 }); // returns age starting from 0 as the first entry
+```
+
+<h3 id="sorting-multiple"><a href="#sorting-multiple">Sorting with Multiple Fields</a></h3>
+
+When sorting with mutiple fields, It is important to understand that the order you send the sort fields will determine the order of priority.
```

Comment:
"It is important to understand..." -> "the order of the sort keys determines what key the MongoDB server sorts by first."